### PR TITLE
"Incipias: Help the Stranded" description

### LINKED ISSUE #10433: Very minor issues with "Incipias"
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

#8949 added a region and race. So far pretty rudimentary but promising. Incompleteness is fine, but...
* effect "star tail hit" declares sound "star tail hit", but the media file is missing.
* Vanilla no-plugins also logs a `Warning: sound "" is referred to, but does not exist.` to the console/errors.txt, though that may have another reason (unless it's the "Strange Ray" having no sound node) (PPS: pity the message points to no location/context).
* Missions "Incipias: Help The Stranded 1+2" can be active in your mission queue, but display no description when selected on the "M" screen.

### Steps to Reproduce

1. Launch game and read console output (or errors.txt)
2. Play the "visit Hui'uc system - visit Quarg in Cura Dic - etc" questline and select one of them in the "your current missions" list

### Expected Behavior

Once new content appears as release, a "virgin" launch should produce zero error output.

### Screenshots

<details>

The terse quest:
image
</details>

### Link to save file

(load, depart, select mission) [Tester T.txt](https://github.com/user-attachments/files/16741287/Tester.T.txt)

### Operating System

Mint 22

### Game Source

Built from source

### Game Version

current master pulled an hour ago

### Additional Information

_No response_

### PR DESCRIPTION
**Pull request by TheGiraffe3**
**Bug Fix**

Fixes #10433.

## Summary
Adds a description to both Incipias: Help the Stranded missions. Fixes a dialog. Uses two quotes instead of one. Removes an unneeded newline for a `source` block.

Creates `empty.wav` for the Archon's hyperdrive so that errors are not thrown.

## Screenshots
None

## Testing Done
None

## Save File
This save file can be used to test these changes:
Use the one from the issue.

### CODE REVIEW

**Comment by TheGiraffe3** [comment]
Will I have to update the copyright for `empty.wav`?

File: copyright [hunk 1]
```diff
@@ -3615,6 +3615,11 @@ Copyright: NASA
 License: public-domain
  Derived from works by NASA/JPL-Caltech/Malin Space Science Systems.
 
+Files:
+ sounds/empty*
```
Comment (TheGiraffe3) [inline copyright:3619]:
```suggestion
 sounds/empty.wav*
```

**Comment by TomGoodIdea** [comment]
I'd prefer the file to be named "null.wav" or "none.wav".

**Comment by tibetiroka** [comment]
Requesting @warp-core due to prior discussions about how to handle this empty sound name case.

**Comment by warp-core** [comment]
I would prefer to change the way hyperspace sounds are handled to not have a default and instead have the current default sounds referred to by the outfits that use them. I'd prefer to know why the current approach was chosen first, though.

As for the changes here, I'd prefer a file name like "no sound" over "null" and I'm not really sure what it means to claim copyright of a sound file that contains no sound.

**Comment by TheGiraffe3** [comment]
> I would prefer to change the way hyperspace sounds are handled to not have a default and instead have the current default sounds referred to by the outfits that use them. I'd prefer to know why the current approach was chosen first, though.

That would be better; however, it requires code changes I cannot do.
I made the no-sound off of https://github.com/endless-sky/endless-sky/issues/10433#issuecomment-2308961527.

**Comment by TomGoodIdea** [comment]
> I'd prefer to know why the current approach was chosen first, though.

Compatibility reasons, probably.

**Comment by warp-core** [comment]
> > I'd prefer to know why the current approach was chosen first, though.
> 
> Compatibility reasons, probably.

If that is the only reason, I'm not sure "plugins don't need to refer to a sound to continue using it" is worth very much here.
If, however, changing the behaviour to require an outfit name a sound to play it is not desirable, an alternative would be to only play the sounds at all in the absence of some attribute, something like "silent jumps" (or perhaps several for all the different types of jump sounds).

**Comment by reticent-rem** [comment]
> > > I'd prefer to know why the current approach was chosen first, though.
> > 
> > 
> > Compatibility reasons, probably.
> 
> If that is the only reason, I'm not sure "plugins don't need to refer to a sound to continue using it" is worth very much here. If, however, changing the behaviour to require an outfit name a sound to play it is not desirable, an alternative would be to only play the sounds at all in the absence of some attribute, something like "silent jumps" (or perhaps several for all the different types of jump sounds).

Beyond compatibility, my thought is that there's probably value in maintaining simplicity with a single silent sound that can be plugged into any existing or new sound effect rather than bespoke `"silent"` attributes for `"jump sound", "hyperdrive in sound", "hyperdrive out sound", "outfit scan sound", etc`, as well as any new ones.

That being said, I feel like I'm missing some knowledge around the empty sound name case so I could be misunderstanding this.

**Comment by RisingLeaf** [comment]
if the archon sound changes could be split off that would be great, cause the other stuff looks fine

**Review by RisingLeaf — APPROVED** [review]

File: data/incipias/incipias first contact.txt [hunk 2]
```diff
@@ -193,6 +193,8 @@ mission "Incipias: The Derelict"
 
 mission "Incipias: Help The Stranded 1"
 	landing
+	name "Go to the Incipias"
+	description "Go to the Incipias planet <planet>, in the <system> system, to see what they have to say about helping a disabled Incipias ship."
```
Comment (tibetiroka) [inline data/incipias/incipias first contact.txt:197]:
```suggestion
	description "Ask the Incipias for help to save one of their ships."
```
The Quarg don't tell you where to go to, and they are quite reluctant to let you go at all. Also, you land on a station, not a planet. And even if you landed on a planet, `<destination>` would be the appropriate way to say it.

File: data/incipias/incipias first contact.txt [hunk 3]
```diff
@@ -311,14 +313,14 @@ mission "Incipias: No Help For The Stranded"
...
 	on offer
 		fail
-		dialog
-			`You identify the stranded alien ship you assisted earlier on your scanners, but with no way to communicate with them you are unable to to help. You have no choice but to leave them to their doom on this stormy world.`
+		dialog `You identify the stranded alien ship you assisted earlier on your scanners, but with no way to communicate with them you are unable to to help. You have no choice but to leave them to their doom on this stormy world.`
 
 mission "Incipias: Help The Stranded 2"
 	landing
+	name "Take Incipias back to <planet>"
+	description "You rescued some Incipias on <origin>. Take them back to <destination>, their home."
```
Comment (tibetiroka) [inline data/incipias/incipias first contact.txt:321]:
```suggestion
	description "Bring the rescued Incipias back home to <destination>"
```

File: data/incipias/incipias first contact.txt [hunk 4]
```diff
@@ -311,14 +313,14 @@ mission "Incipias: No Help For The Stranded"
 			not "Incipias: Help The Stranded 1: offered"
 	on offer
 		fail
-		dialog
-			`You identify the stranded alien ship you assisted earlier on your scanners, but with no way to communicate with them you are unable to to help. You have no choice but to leave them to their doom on this stormy world.`
+		dialog `You identify the stranded alien ship you assisted earlier on your scanners, but with no way to communicate with them you are unable to to help. You have no choice but to leave them to their doom on this stormy world.`
 
 mission "Incipias: Help The Stranded 2"
 	landing
+	name "Take Incipias back to <planet>"
```
Comment (tibetiroka) [inline data/incipias/incipias first contact.txt:320]:
```suggestion
	name "Bring Incipias back to <planet>"
```

File: data/incipias/incipias first contact.txt (hunk 2 above)
Comment (TheGiraffe3) [reply data/incipias/incipias first contact.txt:197]: (but I wanted a comma)

File: data/incipias/incipias first contact.txt [hunk 5]
```diff
@@ -193,6 +193,8 @@ mission "Incipias: The Derelict"
 
 mission "Incipias: Help The Stranded 1"
 	landing
+	name "Go to the Incipias"
```
Comment (Saugia) [inline data/incipias/incipias first contact.txt:196]:
```suggestion
	name "Seek help from the Incipias"
```
"Go to" seems a bit bland.

**Pull request merged**